packets2/willmsgresp_test.go

```diff
@@ -0,0 +1,62 @@
+package packets2
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	pkts "github.com/energomonitor/bisquitt/packets"
+)
+
+func TestWillMsgRespConstructor(t *testing.T) {
+	assert := assert.New(t)
+
+	pkt := NewWillMsgResp(RC_SUCCESS)
```

Comment:
Any reason why the code is changed from `RC_COGNESTION` used in the same test in `packets1`?

(Not that it matters. I just noticed the difference in the `packets1` ↔︎ `packets2` diff.)

Reply:
Good point! The code in `packets1` was improved and I forgot to make the same improvement in `packets2`. `RC_SUCCES` is `0` which is Go's zero value => it's better to test the constructor with any other value.

Fixed.